Develop1

NastyaPilat opened the pull request.

maxpilat committed:
new file:   .coverage
	new file:   __pycache__/__init__.cpython-312.pyc
	new file:   __pycache__/bun.cpython-312.pyc
	new file:   praktikum/__init__.py
	new file:   praktikum/__pycache__/__init__.cpython-312.pyc
	new file:   praktikum/__pycache__/bun.cpython-312.pyc
	new file:   praktikum/__pycache__/burger.cpython-312.pyc
	new file:   praktikum/__pycache__/database.cpython-312.pyc
	new file:   praktikum/__pycache__/ingredient.cpython-312.pyc
	new file:   praktikum/__pycache__/ingredient_types.cpython-312.pyc
	new file:   praktikum/bun.py
	new file:   praktikum/burger.py
	new file:   praktikum/database.py
	new file:   praktikum/ingredient.py
	new file:   praktikum/ingredient_types.py
	new file:   tests/__init__.py
	new file:   tests/__pycache__/__init__.cpython-312.pyc
	new file:   tests/__pycache__/test_bun.cpython-312-pytest-8.2.2.pyc
	new file:   tests/__pycache__/test_burger.cpython-312-pytest-8.2.2.pyc
	new file:   tests/__pycache__/test_database.cpython-312-pytest-8.2.2.pyc
	new file:   tests/__pycache__/test_ingredient.cpython-312-pytest-8.2.2.pyc
	new file:   tests/test_bun.py
	new file:   tests/test_burger.py
	new file:   tests/test_ingredient.py

Commit:
new file:   .gitignore

NastyaPilat committed:
Delete __pycache__ directory

NastyaPilat committed:
Delete tests/__pycache__ directory

Commit:
deleted:    __init__.py
	modified:   praktikum/__pycache__/__init__.cpython-312.pyc
	modified:   praktikum/__pycache__/bun.cpython-312.pyc
	modified:   praktikum/__pycache__/burger.cpython-312.pyc
	modified:   praktikum/__pycache__/ingredient.cpython-312.pyc
	modified:   praktikum/__pycache__/ingredient_types.cpython-312.pyc

Commit:
modified:   tests/test_burger.py

Commit:
new file:   requirements.txt

Commit:
new file:   htmlcov/.gitignore
	new file:   htmlcov/class_index.html
	new file:   htmlcov/coverage_html_cb_497bf287.js
	new file:   htmlcov/favicon_32_cb_58284776.png
	new file:   htmlcov/function_index.html
	new file:   htmlcov/index.html
	new file:   htmlcov/keybd_closed_cb_ce680311.png
	new file:   htmlcov/status.json
	new file:   htmlcov/style_cb_718ce007.css
	new file:   htmlcov/z_c68eb0c7512457e4___init___py.html
	new file:   htmlcov/z_c68eb0c7512457e4_bun_py.html
	new file:   htmlcov/z_c68eb0c7512457e4_burger_py.html
	new file:   htmlcov/z_c68eb0c7512457e4_database_py.html
	new file:   htmlcov/z_c68eb0c7512457e4_ingredient_py.html
	new file:   htmlcov/z_c68eb0c7512457e4_ingredient_types_py.html

klushcheva commented on tests/test_burger.py:
```diff
@@ -0,0 +1,73 @@
+class TestBurger:
+
+    def test_set_buns(self):
+        burger = Burger()
```
Можно лучше: использовать фикстуры для создания объектов

klushcheva reviewed (commented):
Анастасия, хорошо, тут вопросов нет, одно замечание на будущее

Commit:
new file:   conftest.py
	modified:   tests/test_burger.py

Commit:
deleted:    bun.py
	deleted:    burger.py
	deleted:    database.py
	deleted:    ingredient.py
	deleted:    ingredient_types.py
	renamed:    praktikum.py -> praktikum/praktikum.py
